targets/XPlatCppSdk/source/cppsdk/include/playfab/PlayFabBaseModel.h

```diff
@@ -99,12 +99,10 @@ namespace PlayFab
     inline void FromJsonUtilT(const Json::Value& input, time_t& output)
     {
         if (input == Json::Value::null) return;
-        std::string enumStr = input.asString();
+        const std::string timeStr = input.asString();
         tm timeStruct = {};
-        std::istringstream iss(enumStr.c_str());
+        std::istringstream iss(timeStr);
         iss >> std::get_time(&timeStruct, "%Y-%m-%dT%T");
```

Comment:
Should we check what happens if the parse of "iss" fails ? Can this bubble up as an exception in the game ?

Reply:
I think this is a larger discussion. The entire serialization scheme basically tries its best, and defaults to something (hopefully reasonable) if it fails. I'm open to discussing the proposal, but we would need to handle it in ALL deserializations, not just this one. It's certainly not a trivial change, and shouldn't be in this checkin.